cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go

```diff
@@ -224,7 +224,13 @@ func validateKubeConfig(outDir, filename string, config *clientcmdapi.Config) er
 	expectedCtx := config.CurrentContext
 	expectedCluster := config.Contexts[expectedCtx].Cluster
 	currentCtx := currentConfig.CurrentContext
+	if config.Contexts[currentCtx] == nil {
```

Comment:
Should it be `currentConfig`? Also, should we check the `config.Contexts[expectedCtx].Cluster` above?

Reply:
>Should it be currentConfig?  

yeah already sent a change.

> Also, should we check the config.Contexts[expectedCtx].Cluster above

its a file we just generated, so doesn't seem we need to.